README.md

```diff
@@ -229,6 +229,29 @@ It supports the ArchEthic Cryptography rules which are:
   const archethic = require('archethic')
   const index = archethic.getTransactionIndex("00b1d3750edb9381c96b1a975a55b5b4e4fb37bfab104c10b0b6c9a00433ec4646", "https://www.archethic.net")
   // 0
+  ```
+
+    #### getLastTransaction(address, endpoint)
+  Query a node to find the last address of a chain
```

Comment:
I think there is an inconsistency between the name of the function and the description.
Is it to return the last transaction address or the last transaction ?

Reply:
Hi ! 
I updated retrived informations as it returns not only the address